src/app/components/form/form.component.html

```diff
@@ -53,182 +59,232 @@ <h1 i18n>Domain name</h1>
+                          type="button"
+                          (click)="addNewRow('ds_info')"
+                          class="btn btn-success add"
+                          i18n-title
+                          title="Add new DS record">
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                  </button>
+                </div>
               </div>
             </div>
-          </div>
+            <div class="col-auto">
+              <button type="button" class="btn btn-secondary fetchDataFromParent" (click)="fetchDataFromParent('DS')" i18n>Fetch DS from parent zone</button>
+            </div>
+          </fieldset>
 
           <hr>
 
-          <h4 i18n>Delegation Signers (DS records)</h4>
-          <div formArrayName="ds_info">
-            <div class="row mb-2" *ngFor="let ds_info of form.controls.ds_info.controls; let i=index" [formGroupName]="i">
-              <div class="col-md-2">
-                <input formControlName="keytag" class="form-control" i18n-placeholder placeholder="Key Tag"
-                       [class.is-invalid]="ds_info.controls.keytag.invalid && (ds_info.controls.keytag.dirty || ds_info.controls.keytag.touched)">
-                <div class="invalid-feedback" *ngIf="ds_info.controls.keytag.errors?.serverError" >
-                  {{ ds_info.controls.keytag.errors.serverError }}
+          <fieldset>
+            <legend i18n>General</legend>
+            <div class="row">
+              <fieldset aria-describedby="protocol-error" class="col-md-5 form-col">
+                <legend for="profile">Protocol</legend>
+                <div class="protocol" [class.is-invalid]="form.errors?.noProtocol">
+                  <div class="custom-control form-check form-check-inline">
+                    <input type="checkbox" class="form-check-input" formControlName="disable_ipv4" id="disable_protocol_ipv4" [class.is-invalid]="form.errors?.noProtocol || form.controls.disable_ipv4.invalid">
+                    <label i18n class="form-check-label" for="disable_protocol_ipv4">
+                      Disable IPv4
+                    </label>
+                  </div>
+                  <div class="custom-control form-check form-check-inline">
+                    <input type="checkbox" class="form-check-input" formControlName="disable_ipv6" id="disable_protocol_ipv6" [class.is-invalid]="form.errors?.noProtocol || form.controls.disable_ipv6.invalid">
+                    <label i18n class="form-check-label" for="disable_protocol_ipv6">
+                      Disable IPv6
+                    </label>
+                  </div>
                 </div>
-                <div i18n class="invalid-feedback" *ngIf="ds_info.controls.keytag.errors?.required" >
-                  Keytag required
+                <div id="protocol-error" i18n class="invalid-feedback" *ngIf="form.errors?.noProtocol">
+                  Choose at least one protocol
```

Comment:
Suggest: "Choose at least one protocol" --> "Cannot disable both IPv4 and IPv6"

Reply:
Done